extensions/blocks/contact-form/index.js

```diff
@@ -33,7 +33,7 @@ export const settings = {
 	],
 	category: 'jetpack',
 	supports: {
-		reusable: false,
+		reusable: true,
```

Comment:
You could just remove the whole line since this defaults to `true`.

https://developer.wordpress.org/block-editor/developers/block-api/block-registration/#supports-optional